pkg/ring/lifecycler.go

```diff
@@ -63,8 +63,8 @@ type LifecyclerConfig struct {
 	SkipUnregister bool   `yaml:"-"`
 
 	// graveyard for unused flags.
-	UnusedFlag  bool `yaml:"claim_on_rollout,omitempty"` // DEPRECATED - left for backwards-compatibility
-	UnusedFlag2 bool `yaml:"normalise_tokens,omitempty"` // DEPRECATED - left for backwards-compatibility
+	UnusedFlag  bool `yaml:"claim_on_rollout"` // DEPRECATED - left for backwards-compatibility
+	UnusedFlag2 bool `yaml:"normalise_tokens"` // DEPRECATED - left for backwards-compatibility
```

Comment:
I'd consider an exception here - these fields are ignored so could be confusing to emit them.

Reply:
Okay, will reset the change.